.github/workflows/rebuild.yml

```diff
@@ -21,6 +21,13 @@ jobs:
           image-ref: 'docker.io/sonarsource/sonar-scanner-cli'
           exit-code: 1
           ignore-unfixed: true
+      - name: Notify failures on Slack
```

Comment:
I would remove this slack notification here as well. the condition that there is a CVE in the image will trigger a rebuild of the image with updated upstream packages that will hopefully fix this CVE. for vulnerabilities in the application itself we should look at the sonar-scanner-cli repository.
getting a notification whenever there is a upstream package with a vulnerability that will automatically updated during the same run seems not necessary for me